app/workers/podcasts/get_episodes_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Podcasts
-  class GetEpisodesJob < ApplicationJob
-    queue_as :podcasts_get_episodes
+  class GetEpisodesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
 
     def perform(podcast_id:, limit: 1000, force_update: false, feed: Podcasts::Feed)
```

Comment:
You cannot use symbols when [sending arguments to Sidekiq.](https://github.com/mperham/sidekiq/wiki/Best-Practices)

> The arguments you pass to perform_async must be composed of simple JSON datatypes: string, integer, float, boolean, null(nil), array and hash. This means you must not use ruby symbols as arguments.

so these need to be converted to being traditional arguments. Also when you are converting them, if there are any arguments we are not using then feel free to remove them and use the default directly in the job.